### DISCUSSION TITLE
[Office Hours] UI Help for View

### DISCUSSION THREAD

**Discussion by kcallon** [CS342/.github#75] [Office Hours]
### Related Discussion

My current medication view is not great and I would love some tips on how to change the UI to look better. Specifically, I've tried to get the show settings to the very top right. I also want to get the medications listed like in the second photo on the main page, but I'm having trouble adapting it. 
<img width="311" alt="Screenshot 2024-02-22 at 11 26 07 PM" src="https://github.com/CS342/.github/assets/70660419/f9cdb554-a847-4371-9dd3-30b4b37f7e10">
<img width="314" alt="Screenshot 2024-02-22 at 11 27 51 PM" src="https://github.com/CS342/.github/assets/70660419/cc54723d-4e81-4181-8a67-443247952b14">

### Reproduction

var body: some View {
        VStack {
            HStack {
                    Spacer() // Pushes the button to the right
                    Button("Show Settings") {
                        presentSettings.toggle()
                    }
                }
                .padding()
//            Button("Show Settings") {
//                presentSettings.toggle()
//            }
            Text(medicationSettingsViewModel?.description ?? "")
        }
            .sheet(isPresented: $presentSettings) {
                if let medicationSettingsViewModel {
                    NavigationStack {
                        MedicationSettings(isPresented: $presentSettings, medicationSettingsViewModel: medicationSettingsViewModel)
                            .navigationTitle("Medication Settings")
                    }
                } else {
                    ProgressView()
                }
            }
            .task {
                let patientMedications = fhirStore.llmMedications
                self.medicationSettingsViewModel = IntakeMedicationSettingsViewModel(existingMedications: patientMedications)
            }
    }
}. This is my current view, and I thought that the HStack would fix things but am still struggling to adjust this. 

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment]
@kcallon has this issue been resolved in the office hours? If so, can you are some details and maybe a link to your implementation for future students that might have similar questions?
